exporter/opentelemetry-exporter-prometheus/src/opentelemetry/exporter/prometheus/__init__.py

```diff
@@ -175,6 +180,20 @@ def collect(self) -> None:
 
         metric_family_id_metric_family = {}
 
+        if len(self._metrics_datas) != 0:
```

Comment:
You can write this as `if len(self._metrics_datas):`.

Reply:
Thanks for the suggestion!